src/components/Card/styled.js

```diff
@@ -1,8 +1,10 @@
 import styled from 'styled-components';
 
-const StyledCard = styled.section`
+const StyledCard = styled.button`
```

Comment:
That's confusing: Is it a Card or a Button? I think there is a big difference, so if it's a button, rename your styled component as well.